x-pack/solutions/security/plugins/security_solution/public/app/actions/toggle_column/cell_action/toggle_column.ts

```diff
@@ -67,6 +67,11 @@ export const createToggleColumnCellActionFactory = createCellActionFactory(
         return;
       }
 
+      if (metadata.toggleColumnRef?.current) {
```

Comment:
should not be executed when action is running in any scope other than alerts table. Will be great if you can add a comment as well that this only applicable to alerts table.


```suggestion
      if (metadata.toggleColumnRef?.current && scopeId == TableId.alertsOnAlertsPage  ) {
```

Reply:
Done! ✅